### PR TITLE
refactor: make ropsten config more consistent with other markets

### PR DESCRIPTION
**Pull request by defispartan**
- Display testnet icon
- Make market names consistent
- Change ordering to place ETH testnets together

Old

oldswitcher

New

newswitcher

### CODE REVIEW

**Review by foodaka — DISMISSED** [review]

**Comment by foodaka** [comment]
@defispartan one thing a user reported an issue with the state of the dropdowns. 
Screenshot 2022-06-08 at 13 26 31

**Comment by defispartan** [comment]
The top right is network that your wallet is connected to which can be different than the market you're currently viewing

**Comment by defispartan** [comment]
I am going to make one change to this PR though, @iamanastasia added a design for how the network names should be displayed which I'll add here

File: src/components/MarketSwitcher.tsx [hunk 1]
```diff
@@ -248,19 +252,25 @@ export const MarketSwitcher = () => {
...
+                testChainName={marketNaming.testChainName}
               />
-              <ListItemText sx={{ mr: 3 }}>
-                {getMarketHelpData(market.marketTitle).name} {market.isFork ? 'Fork' : ''}
+              <ListItemText sx={{ mr: 0 }}>
+                {marketNaming.name} {market.isFork ? 'Fork' : ''}
               </ListItemText>
 
-              {currentMarket === marketId && (
+              {/*               {currentMarket === marketId && (
```
Comment (grothem) [inline src/components/MarketSwitcher.tsx:261]: Should we just delete this?

**Review by grothem — DISMISSED** [review]

File: src/components/MarketSwitcher.tsx (hunk 1 above)
Comment (defispartan) [reply src/components/MarketSwitcher.tsx:261]: Yep I'll remove it

**Review by grothem — APPROVED** [review]

**Review by foodaka — APPROVED** [review]

**Pull request merged**